qiskit/extensions/quantum_initializer/initializer.py

```diff
@@ -363,7 +363,83 @@ def validate_parameter(self, parameter):
 
 
 def initialize(self, params, qubits=None):
-    """Apply initialize to circuit."""
+    """Qubit initializalition is done by appending instructions to the quantum circuit (by
+    calling Initialize(params)) and the qubits we wish to iniatilize. Note that the
+    qubits are first set to `|0>` and then the desired state is achieved by applying
+    a state preparing unitary.
+
+    Args:
+        params (str or list):
+            * list: vector of complex amplitudes to initialize to.
```

Comment:
#5418 just added an `int` param to `initialize` and `Initialize`. It works like the `from_int` in `Statevector`.

Reply:
Fixed in d483846.